pkg/operator/k8sutil/service.go

```diff
@@ -92,3 +98,72 @@ func ParseServiceType(serviceString string) v1.ServiceType {
 	}
 	return v1.ServiceType("")
 }
+
+// ExportService exports a k8s service using Multi Cluster Service API client
+func ExportService(ctx context.Context, client *mcsv1Client.Clientset, service *v1.Service) error {
+	serviceExport := &mcsv1a1.ServiceExport{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            service.Name,
+			Namespace:       service.Namespace,
+			OwnerReferences: service.GetOwnerReferences(),
+		},
+	}
+
+	_, err := client.MulticlusterV1alpha1().ServiceExports(service.Namespace).Create(ctx, serviceExport, metav1.CreateOptions{})
```

Comment:
If we're using v1alpha1 it's at risk for a changing API in the future. Any clue on when they are trying to go to beta?